File: livingrimoire start here/LivinGrimoire java/src/Auxiliary_Modules/AXNeuroSama.java
package Auxiliary_Modules;

public class AXNeuroSama {
    private final Responder nyaa = new Responder(" heart", " heart", " wink", " heart heart heart");
    private final DrawRnd rnd = new DrawRnd();
    private final int rate;

    public AXNeuroSama(int rate) {
        // the higher the rate the less likely to decorate outputs
        this.rate = rate;
    }

    public String decorate(String output){
        if (output.isEmpty()){
            return output;
        }
        if(rnd.getSimpleRNDNum(rate) == 0){
            return output + nyaa.getAResponse();
        }
        return output;
    }
}

File: livingrimoire start here/LivinGrimoire Kotlin/kotlin/auxiliary_modules/AXNeuroSama.kt
package auxiliary_modules

class AXNeuroSama    // the higher the rate the less likely to decorate outputs
    (private val rate: Int) {
    private val nyaa = Responder(" heart", " heart", " wink", " heart heart heart")
    private val rnd = DrawRnd()
    fun decorate(output: String): String {
        if (output.isEmpty()) {
            return output
        }
        return if (rnd.getSimpleRNDNum(rate) == 0) {
            output + nyaa.aResponse
        } else output
    }
}